src/main/main.ts

```diff
@@ -635,7 +635,7 @@ ipcMain.on(
 
         const globalMediaKeysEnabled = store.get('global_media_hotkeys') as boolean;
 
-        if (globalMediaKeysEnabled) {
+        if (globalMediaKeysEnabled !== false) {
```

Comment:
Rather than changing the condition, a potentially more clear option would be `store.get('global_media_hotkeys', true) as boolean` (`true` being the default when not defined).

Reply:
I didn't know that was a thing, I'll definitely use that instead to clear up the code.